config/src/keyassignment.rs

```diff
@@ -673,6 +673,7 @@ pub enum CopyModeAssignment {
     PageUp,
     PageDown,
     Close,
+    CloseWithoutClear,
```

Comment:
I don't think `Clear` is the correct way to describe what happens.

How about we tweak this PR:

* `Close` - closes without changing the viewport. This is a break in backwards compatibility for folks that are using this assignment in their configs, but I think there are relatively few people that are doing that so the impact should be small
* Change the `copy_key_table` function (https://github.com/wez/wezterm/blob/6e85ba49fdd0a550d1806289f8410b8b7047bf48/wezterm-gui/src/overlay/copy.rs#L1577)  to use something like the following instead of `Close`:

```rust
fn close_and_scroll_to_bottom() -> KeyAssignment {
  KeyAssignment::Multiple(vec![
     KeyAssignment::CopyMode(CopyModeAssignment::Close),
     KeyAssignment::ScrollToBottom
  ]) 
}

...

(
    WKeyCode::Char('\x1b'),
    Modifiers::NONE,
    close_and_scroll_to_bottom(),
),
```

this way it should be easier to reason about and compose key assignment actions, without having to pre-define actions for every combination.

Reply:
I used "Clear" since the operation being skipped is "ClearViewport" -- maybe "ResetViewport" is more accurate vs. "ScrollToBottom"

Splitting up into two composed key assignment actions sounds reasonable; the reason I went with keeping "Close" unchanged is to not break people's existing configs in case they remapped it, but I have no strong opinion here.

Reply:
Done (as ResetViewport, and keybindings without a helper function for consistency with surrounding code).